deteriorate.lua

```diff
@@ -35,11 +36,48 @@ local function is_valid_clothing(item)
             and item.wear > 0
 end
 
-local function is_valid_corpse(item)
-    return not item.flags.dead_dwarf
+local function is_not_useable(opts, item)
+    local not_useable =
+        not(opts.useable and (
+                not item.corpse_flags.unbutchered and (
+                    item.corpse_flags.bone or
+                    item.corpse_flags.horn or
+                    item.corpse_flags.leather or
+                    item.corpse_flags.skull1 or
+                    item.corpse_flags.skull2 or
```

Comment:
what is the difference between skull1 and skull2?

Reply:
See some the item_body_component data below for an example.
When a body has been butchered (separated), skull1 is true for nervous tissue and skull2 is true for skulls.
However when the bodies are not butchered, then skull2 is false and skull1 is true. Initially I thought skull1 meant nervous tissue and was going to exclude it, but as my data is not exhaustive and the name is skull, I've left it in.
There are some flags in the caste raws (NOSKULL, NOSKIN, NOBONES) which I suspect play as part in the butchering process, but haven't found the any code which refers to them yet.
`id                                     Description  unbutchered bone   skull1 skull2  sep_part
376359                          stray alpaca skull  false       false  false   true   false
377190               stray water buffalo cow skull  false       false  false   true   false
378832                          stray alpaca skull  false       false  false   true   false
468655     stray water buffalo calf nervous tissue  false       false   true  false    true
469069     stray water buffalo calf nervous tissue  false       false   true  false    true
472516                         stray yak cow skull  false       false  false   true   false
476342                         stray yak cow skull  false       false  false   true   false
476344                stray yak cow nervous tissue  false       false   true  false    true
476558                        stray yak calf skull  false       false  false   true   false
476560               stray yak calf nervous tissue  false       false   true  false    true
477147                           troglodyte corpse   true        true   true  false    true
477415   queen ant woman blowgunner mangled corpse   true       false   true  false    true
477885                 Xuspgas Akuksos's left hand   true        true   true  false    true
477980                      Bosa Ugsnorurar's neck   true        true   true  false    true
478297                      giant cave toad corpse   true        true   true  false    true
478349       drone ant man spearman mangled corpse   true       false   true  false    true
478837 soldier ant woman blowgunner mangled corpse   true       false   true  false    true
478853               drone ant man spearman corpse   true       false   true  false    true
477904                         Nako Atuzugu's head   true       false   false   true  false    true`

I'm happy to exclude skull1 if you agree that it means nervous tissue, since that stuff isn't useful.

Reply:
from that list, I don't see any instances of skull1==true where the part is usable, do you? I vote for removing it.

Reply:
Done.